### PR TITLE
Create kachra.md

### PR DESCRIPTION
**Pull request by nvjkmr**

### CODE REVIEW

**Comment by nvjkmr** [comment]
Kya haal hai bro? Sab kachra kyun hai?

**Review by nvjkmr — COMMENTED** [review]
koko

**Review by nvjkmr — COMMENTED** [review]
Something again

**Comment by kartikag1** [comment]
defsghj

**Comment by kartikag1** [comment]
rwgfvbn

**Review by kartikag1 — COMMENTED** [review]
23ewrgt

**Review by nvjkmr — COMMENTED** [review]
tatty

**Comment by nvjkmr** [comment]
tatty nahi. potty

**Comment by kartikag1** [comment]
lkjhj

**Review by kartikag1 — COMMENTED** [review]
njbhgfgdxfzsxcfgvhbjnkml,

**Review by kartikag1 — CHANGES_REQUESTED** [review]
mknjmb

**Review by kartikag1 — APPROVED** [review]

File: kachra.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# sab kachra hai
+
```
Comment (kartikag1) [inline kachra.md:2]: ab aaja please :'(

Comment (kartikag1) [reply kachra.md:2]: please aaaajaaa

Comment (kartikag1) [reply kachra.md:2]: please aaaajaaa

Comment (nvjkmr) [reply kachra.md:2]: tatty ayega

Comment (kartikag1) [reply kachra.md:2]: please aaaajaaaplease aaaajaaaplease aaaajaaaplease aaaajaaaplease aaaajaaaplease aaaajaaa

Comment (kartikag1) [reply kachra.md:2]: please aaaajaaa

Comment (kartikag1) [reply kachra.md:2]: 2qer

Comment (nvjkmr) [reply kachra.md:2]: tatty karega

Comment (nvjkmr) [reply kachra.md:2]: aa gaya

Comment (nvjkmr) [reply kachra.md:2]: tatty ayega ayega... ayegaa

**Comment by nvjkmr** [comment]
aayayayayayayaa

**Review by nvjkmr — COMMENTED** [review]
karna toh hai.... lekin hua nahi...

**Pull request merged**